src/pages/public_registration/action_accordions/SupportAccordion.tsx

```diff
@@ -45,10 +69,21 @@ export const SupportAccordion = ({ registration, supportTicket, addMessage, refe
       elevation={3}>
       <AccordionSummary sx={{ fontWeight: 700 }} expandIcon={<ExpandMoreIcon fontSize="large" />}>
         {t('my_page.messages.types.GeneralSupportCase')}
+        {supportTicket && ` - ${t(`my_page.messages.ticket_types.${supportTicket.status}`)}`}
       </AccordionSummary>
       <AccordionDetails sx={{ display: 'flex', flexDirection: 'column', gap: '1rem' }}>
         {supportTicket && (
           <>
+            {userIsCurator && !currentPath.startsWith('/registration') && (
```

Comment:
Siden det er en sjekk på `userIsCurator` her antar jeg at dette bare skal være lov når man er på `/tasks`? (Og at creator ikke skal kunne gjøre dette)
```suggestion
            {userIsCurator && window.location.pathname.startsWith(UrlPathTemplate.Tasks) && (
```

Reply:
Da er egentlig sjekken på `userIsCurator` overflødig også, men skader ikke å ta den med heller for så vidt da